templates/agenda_case_item/presentation.html.twig

```diff
@@ -0,0 +1,28 @@
+{% extends 'layout-with-agenda-case-item-submenu.html.twig' %}
+
+{% trans_default_domain 'agenda_item' %}
+
+{% block title %}{% trans %}Case presentation{% endtrans %}{% endblock %}
+
+{% block header %}
+    <div class="row">
+        <div class="col">
+            <h1>{% trans %}Case presentation{% endtrans %}</h1>
+            <p class="text-muted"><small>{% trans %}Create or update case presentation{% endtrans %}</small></p>
+        </div>
+        <div class="col-auto">
+            {#            <a name="" id="" class="btn btn-secondary" href="{{ url('agenda_show', {id: agenda.id}) }}" role="button">{% trans %}Cancel{% endtrans %}</a>#}
+        </div>
+    </div>
+{% endblock %}
+
+{% block content %}
+
+    <main role="main" class="col-12 mt-3">
+        {{ form(case_presentation_form) }}
+    </main>
+{% endblock %}
+
+{% block javascripts %}
+    {{ parent() }}
+{% endblock %}
```

Comment:
Isn't this already included in the parent?

Reply:
Removed redundant `parent()` call.